src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -218,35 +190,35 @@ void InitInteriorConvectionCoeffs(EnergyPlusData &state,
             allocated(state.dataLoopNodes->Node)) {
             state.dataConvectionCoefficient->NodeCheck = false;
             for (int ZoneNum = 1; ZoneNum <= state.dataGlobal->NumOfZones; ++ZoneNum) {
-                if (Zone(ZoneNum).InsideConvectionAlgo != ConvectionConstants::HcInt_CeilingDiffuser) continue;
```

Comment:
There is some sense in `ConvectionCoefficients` and `ConvectionConstants` being separate files, but there is no sense in making them separate namespaces.

Reply:
Not in scope but I think !state.dataGlobal->DoingSizing would work at line 189 instead of using the 2 other flags.

Reply:
??